src/prompt/prompt.js

```diff
@@ -151,15 +154,18 @@ export async function promptForEmailAndPasswordApiKey() {
   const usernamePasswordQuestions = [{
     name: 'email',
     type: 'input',
-    message: 'Email:'
+    message: 'Email:',
+    validate: value => credentialValidator.validateEmail(value)
```

Comment:
Same with all three of these in this file. You don't need to create an anonymous function to call the callback function.